source/2020-02-14-the-ember-times-issue-135.md

```diff
@@ -48,13 +49,16 @@ Be sure to be on the lookout for more updates from the Ember Times team as more
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
-
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [There is no party like an Octane party 🎊](https://changelog.com/jsparty/114)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+On the 6th of February [Jen Weber @jenweber](https://github.com/jenweber) and [Chris Manson @mansona
+](https://github.com/mansona) were live on the [JS Party podcast](https://changelog.com/jsparty). This party describes themselves as: "A community celebration of JavaScript and the web.". They record on Thursdays at 1pm Eastern / 10am Pacific if you want to tune in live. 
```

Comment:
Although a party has many people, I think the word is still considered singular.
```suggestion
](https://github.com/mansona) were live on the [JS Party podcast](https://changelog.com/jsparty). This party describes itself as: "A community celebration of JavaScript and the web." They record on Thursdays at 1pm Eastern / 10am Pacific if you want to tune in live. 
```